File: ce/customerengagement/on-premises/snippets/csharp/CRMV8/solutions/cs/getsolutiondependencies4.cs
// <snippetgetsolutiondependencies4>


 ExportSolutionRequest exportRequest = new ExportSolutionRequest
 {
     Managed = true,
     SolutionName = _primarySolutionName
 };
 ExportSolutionResponse exportResponse =
     (ExportSolutionResponse)_serviceProxy.Execute(exportRequest);

// </snippetgetsolutiondependencies4>

File: ce/customerengagement/on-premises/snippets/visualbasic/CRMV8/solutionsvb/vb/getsolutiondependencies4.vb
' <snippetgetsolutiondependencies4>


 Dim exportRequest As ExportSolutionRequest =
  New ExportSolutionRequest With {
   .Managed = True,
   .SolutionName = _primarySolutionName
  }
 Dim exportResponse As ExportSolutionResponse =
  CType(_serviceProxy.Execute(exportRequest), ExportSolutionResponse)

' </snippetgetsolutiondependencies4>